polling/infrequent/worker/main.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"log"
+
+	"github.com/temporalio/samples-go/polling"
```

Comment:
No need for blank lines between the non-stdlib imports. This should be grouped with the other one further down IMO.

Reply:
Interesting, `goimports -w .` do not fix the issue neither does running go Imports from GoLand.